### PR TITLE
Fixes of small typos, proposals of small improvements

### PR DESCRIPTION
**Pull request by expln**
I read "3.3 Proof: Principle of the syllogism (syl)" and noticed a few places where changes might be needed. Please review my proposals.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1941,7 +1941,7 @@ Let's eliminate this justification:
 > icon <img width="16" height="16" src="delete.svg" alt="delete"> (delete)
 > to delete the justification.
 
-Let's unify and see what happens:
+Let's unify and see what happens (make sure all the steps are unselected before proceeding):
```
Comment (expln) [inline docs/index.md:1944]: I executed “Show completed proof” previously, so one step remained selected in my setup.

Comment (david-a-wheeler) [reply docs/index.md:1944]: Good idea!

File: docs/index.md [hunk 2]
```diff
@@ -2105,9 +2105,9 @@ instead of completing this step:
 
 > Click on the
 > icon <img width="16" height="16" src="replacement.svg" alt="global substitution"> (global substitution<!-- ‡ -->).
-> In "Replace what" use the value &amp;W1 and in
-> "replace with" use the value `( ph -> ( ps -> ch ) )` and then press Return.
-> Click "Find Substitution<!-- ‡ -->" button; metamath-lamp will
+> In "Match what" use the value &amp;W1 and in
+> "Match with" use the value `( ph -> ( ps -> ch ) )` and then press Return
+> (or click "Find Substitution<!-- ‡ -->" button). Metamath-lamp will
```
Comment (david-a-wheeler) [inline docs/index.md:2110]: "or click" should be "or click the".

**Comment by david-a-wheeler** [comment]
Looks great! One small missing "the" (see above).

File: docs/index.md (hunk 2 above)
Comment (expln) [reply docs/index.md]: Fixed. Thanks!

**Comment by david-a-wheeler** [comment]
Awesome! Thanks!

**Review by david-a-wheeler — COMMENTED** [review]
Awesome! Thanks!

**Pull request merged**